### PR TITLE
Replace "Be part of Dryad" with twitter feed

### PR DESCRIPTION
**Pull request by ryscher**
On the homepage, replace the "Be part of Dryad" section with a feed of twitter content.

This needs to be reviewed by Elizabeth (and perhaps others). Before final integration, the "Be part of Dryad" content needs to be placed on a new page.

### CODE REVIEW

**Comment by elizabethdryad** [comment]
Can I see it on dev?

**Comment by ryscher** [comment]
Need to correct the color for the links -- the current color doesn't seem to be supported by the twitter widget.

**Comment by elizabethdryad** [comment]
Change box header to "Latest from @datadryad"

**Comment by elizabethdryad** [comment]
Okay to go ahead with this, with header change -- demo was reviewed/approved, and Elizabeth will repurpose "Be part of Dryad" content in another PR.

**Comment by elizabethdryad** [comment]
Can be launched separately from or in conjunction with https://github.com/datadryad/dryad-repo/pull/896

File: dspace/modules/xmlui/src/main/webapp/themes/Mirage/Mirage.xsl [hunk 1]
```diff
@@ -576,7 +548,7 @@
     <xsl:template match="dri:options/dri:list[@n='DryadConnect']" priority="3">
       <div class="NOT-simple-box">
         <!-- START CONNECT  -->
-        <h1 class="ds-div-head ds_connect_with_dryad_head" id="ds_connect_with_dryad_head">Be part of Dryad
+        <h1 class="ds-div-head ds_connect_with_dryad_head" id="ds_connect_with_dryad_head">Tweets from Dryad
```
Comment (daisieh) [inline dspace/modules/xmlui/src/main/webapp/themes/Mirage/Mirage.xsl:551]: I think that this one is causing the sidebar to say "Tweets from Dryad" even on pages where the sidebar doesn't have tweets, like membershipOverview.

File: dspace/modules/xmlui/src/main/webapp/themes/Mirage/Mirage.xsl [hunk 2]
```diff
@@ -317,41 +317,13 @@
 
             <!-- START CONNECT  -->
             <div class="home-col-2" style="clear: right;">
-                <h1 class="ds-div-head ds_connect_with_dryad_head" id="ds_connect_with_dryad_head">Be part of Dryad
+                <h1 class="ds-div-head ds_connect_with_dryad_head" id="ds_connect_with_dryad_head">Tweets from Dryad
```
Comment (daisieh) [inline dspace/modules/xmlui/src/main/webapp/themes/Mirage/Mirage.xsl:320]: please change wording to "Latest from @datadryad" as per @elizabethdryad's comment.

**Comment by elizabethdryad** [comment]
Holding on this until we have a plan for representing membership, integration AND payment plans on the homepage

**Comment by elizabethdryad** [comment]
Okay to proceed now, in conjunction with https://github.com/datadryad/dryad-repo/pull/905

**Comment by daisieh** [comment]
@ryscher, when you get a chance, can you address the issues I pointed out in the line notes?

**Comment by daisieh** [comment]
I replaced this PR with #970. Closing.

**Pull request closed without merging**